src/python-et-asyncio-bonheur.rst

```diff
@@ -272,16 +272,17 @@ Soit environ 4 secondes, 0.5 seconde par image.
 
 Plusieurs remarques :
 
-- dans le cas du code asynchrone, il faut faire bien attention d'utiliser des
-  appels non bloquants uniquement. On utilise donc `aiohttp` pour récupérer la
-  page et l'image, puis faire les miniatures (en utilisant
+- dans le cas du code asynchrone, il faut faire bien attention d'utiliser
+  uniquement des appels non bloquants. On utilise donc `aiohttp` pour récupérer
+  la page et l'image, puis faire les miniatures (en utilisant
   ``loop.run_in_executor``).
 - plus le code dans ``compute_thumbnails`` sera gourmand en CPU, et sera donc
   long a exécuter, plus on gagnera en performance sur la version asynchrone par
   rapport à la version synchrone, le temps de CPU étant "masqué" par le temps
   du téléchargement des pages et des images.
-- le code asynchrone est plus long et complexe, et nécessite de penser le
-  programme différemment.
+- le code asynchrone est plus long, plus complexe, et nécessite de penser le
+  programme différemment. **(tu mettrais aussi un petit mot sur le debug, bien
+  merdique - ou pas ^^?)**
```

Comment:
ah, oui, pourquoi pas un bullet point supplémentaire, avec peut-être un lien vers https://pymotw.com/3/asyncio/debugging.html et/ou vers https://docs.python.org/3/library/asyncio-dev.html#debug-mode-of-asyncio